perk-store-services/src/main/java/org/exoplatform/perkstore/model/constant/ProductOrderType.java

```diff
@@ -0,0 +1,23 @@
+package org.exoplatform.perkstore.model.constant;
+
+public enum ProductOrderType {
+  ALL("ALL","all"),
+  SEND("SEND","send"),
```

Comment:
We need to use the adjective 
```suggestion
  SENT("SENT","send"),
```